ni/test/test_github.py

```diff
@@ -78,7 +79,16 @@ def test_ping(self):
                                                    util.FakeRequest(payload)))
         self.assertEqual(cm.exception.response.status, 200)
 
-    def test_process_skipping(self):
+    def test_verify_signature(self):
+        data = b'the-knights-who-say-ni'
+        secret_token = 'some_secret_token'
+        expected = '0b63e1a89ba1a9d18b4b19749431e244c93c7873'
```

Comment:
In case reactions don't trigger a notification, you're right in assuming I'm after the function to generate the hash.